src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/variadic_split.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+typedef std::tuple<std::vector<size_t>,      // Num splits
+                   int64_t,                  // Axis
+                   ov::element::Type,        // Model type
+                   std::vector<InputShape>,  // Input shapes
```

Comment:
Where is dynamic shapes? We supported dynamism In new base class

Reply:
using InputShape = std::pair<ov::PartialShape, std::vector<ov::Shape>>;
So this is what required